src/vcpkg/commands.generate-message-map.cpp

```diff
@@ -39,13 +39,18 @@ namespace vcpkg::Commands
 {
     static constexpr StringLiteral OPTION_ALLOW_BAD_COMMENTS = "allow-incorrect-comments";
     static constexpr StringLiteral OPTION_NO_ALLOW_BAD_COMMENTS = "no-allow-incorrect-comments";
+    static constexpr StringLiteral OPTION_OUTPUT_COMMENTS = "output-comments";
```

Comment:
Why do we need this option? It seems to be a bit redundant. To be explicit?

Reply:
for all options with `no-[blah]`, we also want to add a `[blah]` to support exactly that - being explicit.